bitnami/redis/templates/prometheusrule.yaml

```diff
@@ -9,13 +9,10 @@ kind: PrometheusRule
 metadata:
   name: {{ template "common.names.fullname" . }}
   namespace: {{ default .Release.Namespace .Values.metrics.prometheusRule.namespace | quote }}
-  labels: {{- include "common.labels.standard" . | nindent 4 }}
+  labels: {{- include "common.labels.standard" ( dict "customLabels" .Values.commonLabels "context" $ ) | nindent 4 }}
     {{- if .Values.metrics.prometheusRule.additionalLabels }}
```

Comment:
I'm only merging when there's a `.Values.XXX.labels` parameter and skipping it when the parameter name is `.Values.XXX.additionalLabels` since the naming suggests they should be added (not merged)